packages/extension-chakra-storefront/src/pages/checkout/partials/shipping-address-selection.jsx

```diff
@@ -154,7 +154,19 @@ const ShippingAddressSelection = ({
 
     useEffect(() => {
         if (isBillingAddress) {
-            form.reset({...selectedAddress})
+            // For billing address, use proper default values if no selectedAddress
+            const defaultValues = selectedAddress ? {...selectedAddress} : {
```

Comment:
This is need because countryCode has a default value set to US

Reply:
I'll remove this bit since by the time the user is setting their billing address (in step 4 of checkout), they would have already set a shipping address (in step 2 of checkout) so a selected address should exist.

And if not, I don't know if we want to hard code 'US' here.

Reply:
After discussion, we will add the 'US' default here as it is also hard coded in other areas.
We should create a follow up ticket to clean up the defaults.